server/router/server.go

```diff
@@ -188,6 +188,7 @@ func NewRouter(_ context.Context, h models.HandlerInterface, port int, g http.Ha
 	gMux.Handle("/api/meshmodels/models/{model}", h.ProviderMiddleware(h.AuthMiddleware(http.HandlerFunc(h.GetMeshmodelModelsByName), models.NoAuth))).Methods("GET")
 
 	gMux.Handle("/api/meshmodels/registrants", h.ProviderMiddleware(h.AuthMiddleware(http.HandlerFunc(h.GetMeshmodelRegistrants), models.NoAuth))).Methods("GET")
+	gMux.Handle("/api/meshmodels/nonRegisterEntity", h.ProviderMiddleware(h.AuthMiddleware(http.HandlerFunc(h.NonRegisterEntity), models.NoAuth))).Methods("POST")
```

Comment:
What's this? It doesn't comply with the current REST API's design.

Reply:
Will follow the current REST API's design I got the resource will start working on it right away